plugin/src/main/java/Deps.kt

```diff
@@ -401,12 +409,17 @@ object google {
         androidx.recyclerview,
         androidx.transition
     )
-}
-
-object javax {
-    val inject = deps(
+    private val inject = deps(
         "javax.inject:javax.inject:1".dep
     )
+    val dagger = deps(
```

Comment:
use transitive deps or supply full list :)

Reply:
That is the great question! Do you have agreement about it? My personal preference to have full list

Reply:
my personal preference is to source an entire list of dependencies but currently, it's a time-consuming process which does not contribute much to MVP we are building. So for now I'm ok with transitive deps, one to the items in roadmap is to build some tool to generate the list

Reply:
yea, we use transitive deps only for faster delivery MVP functionality now